src/map/unit.hpp

```diff
@@ -46,6 +46,7 @@ struct unit_data {
 	uint8 dir;
 	unsigned char walk_count;
 	unsigned char target_count;
+	uint8 body_size;
```

Comment:
if the range is 1~400, a uint8 is not big enough to fit this, you would need uint16.